include/utility/json_utility.hpp

```diff
@@ -550,5 +550,59 @@ inline std::string getFruPathFromJson(const nlohmann::json& i_sysCfgJsonObj,
     }
     return std::string();
 }
+
+/**
+ * @brief Get inventory object path from system config JSON.
+ *
+ * Given either D-bus inventory path/FRU EEPROM path/redundant EEPROM path,
+ * this API returns D-bus inventory object path if present in JSON.
+ *
+ * @param[in] i_sysCfgJsonObj - System config JSON object
+ * @param[in] i_vpdPath - Path to where VPD is stored.
+ *
+ * @throw std::runtime_error.
+ *
+ * @return On success return valid path, on failure return empty string.
```

Comment:
@return On success returns valid path, on failure returns empty string or an exception.
OR
@return On success a valid path is returned, on failure an empty string is returned or an exception is thrown.

Reply:
done